src/Compilers/CSharp/Portable/Symbols/TypeSymbol.cs

```diff
@@ -888,17 +907,21 @@ private static Symbol ComputeImplementationForInterfaceMember(Symbol interfaceMe
             if ((object)implicitImpl == null && seenTypeDeclaringInterface)
             {
                 // Check for default interface implementations
-                if (!interfaceMember.IsAbstract)
-                {
-                    implicitImpl = interfaceMember;
-                }
+                implicitImpl = FindMostSpecificImplementation(interfaceMember, implementingType, ref useSiteDiagnostics, diagnostics);
+            }
+
+#if !DEBUG
```

Comment:
To force code path coverage for debug builds, yet avoid calculating location in Release when it is unnecessary. I'll add a comment.